charts/citylens/values.yaml

```
@@ -1234,6 +1250,12 @@ routes:
     oidcMobileAuthorizationEnabled: true
     proxyNaviKeyEnabled: true
     coverageRatioEnabled: true
+    tasksAssetEnabled: false
+
+  tasksAssetSettings:
+    systemAccessToken: ''
+    plannedTasksAssetId: 'user_dyn_citylens_planned_tasks'
+    completedTasksAssetId: 'user_dyn_citylens_completed_tasks'
```

здесь можно указать другие id ассетов или они где-то захардкожены?
```suggestion
    plannedTasksAssetId: user_dyn_citylens_planned_tasks
    completedTasksAssetId: user_dyn_citylens_completed_tasks
```

---

Можно указать другие айди